lib/pbench/cli/agent/commands/stddev.py

```diff
@@ -0,0 +1,28 @@
+"""
+pbench-avg-stddev
+
+
+input: a bunch of numbers on cmd line
+output: an average (mean), sample standard deviation, sample standard deviation
+percent, and which input was closest to the average.
+
+nargs = -1, accepts an unlimited number of arguments
+"""
+
+import click
+
+import statistics
+
+
+@click.command()
+@click.argument("values", nargs=-1, type=click.FLOAT)
+def main(values):
+    values = list(values)
+    avg = statistics.mean(value)
+    stddev = statistics.stdev(value)
+    stddevpct = 100 * stddev / avg
+
+    values = np.array(values)
+    closest_index = min(range(len(value)), key=lambda i: abs(value[i]-avg))
```

Comment:
Huh? Where does `np.array` come from since you dropped numpy? Also, you have `values` but in the search you're using `value`.

Reply:
Trigger happy, fixed